.circleci/config.yml

```diff
@@ -2,49 +2,30 @@ version: 2.1
 
 orbs:
   github-maven-deploy: github-maven-deploy/github-maven-deploy@1.0.5
```

Comment:
This is still needed if you use ours?

Reply:
Yeah! from my understand. It's in the R repo i'm using as my main reference: https://github.com/sonatype-nexus-community/nexus-repository-r/blob/292230836d77aa8445e70ce81793fcdda0395de1/.circleci/config.yml#L4

Reply:
@allenhsieh is correct-we still use Benny’s build-and-test job

Reply:
For some wild reason I thought the new one `extend`ed the previous one, good to know!